flask/app/variants.py

```diff
@@ -196,7 +202,8 @@ def summary(type: str):
             .contains_eager(models.Analysis.datasets)
             .contains_eager(models.Dataset.tissue_sample)
             .contains_eager(models.TissueSample.participant)
-            .contains_eager(models.Participant.family)
+            .contains_eager(models.Participant.family),
+            contains_eager(models.Gene.aliases.of_type(alias_subquery)),
```

Comment:
The common name should be serialized for the variant endpoint, around lines ~255